docs/manual/working/scalaGuide/main/sql/slick/code/DI.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2011-2015 Typesafe Inc. <http://www.typesafe.com>
+ * Copyright (C) 2011-2019 Lightbend Inc. <https://www.lightbend.com>
```

Comment:
Could you please remove the year like we did for Play projects?

```scala
/*
 * Copyright (C) Lightbend Inc. <https://www.lightbend.com>
 */
```

Reply:
Done.